import-service/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "jest",
+    "invoke": "sls invoke local --function",
+    "deploy": "sls deploy",
+    "deploy:fn": "sls deploy function -f"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.792.0",
```

Comment:
we can move it to `devDependencies ` to reduce bundle size

Reply:
@Ilya-Valasiuk 
`aws-sdk` package is used inside lambda functions as part of main functionality. Could you please explain why it can be moved into`devDependencies` as it's required in production build?